packages/sitecore-jss-angular/package.json

```diff
@@ -4,7 +4,7 @@
   "description": "",
   "scripts": {
     "build": "ng-packagr -p ng-package.json",
-    "lint": "eslint --fix ./src/**/*.ts",
+    "lint": "eslint ./src/**/*.ts",
```

Comment:
This fixes an (unrelated issue) we were having with CI broken builds - where the [auto fix was causing surprise source file changes](https://github.com/Sitecore/jss/pull/529/commits/eb0d952c486f048f10c087f1fe17fe22edf9d764). This has bit us a few times now since we added linting to the CI pipeline.

Sorry, should have probably created a separate PR for that change, but decided to sneak it in here :)